File: solutions/Question 3898/Solution1.c
/*
 * Problem: 3898. Find the Degree of Each Vertex
 * Difficulty: Easy
 * Topics: Array, Graph Theory, Matrix
 * URL: https://leetcode.com/problems/find-the-degree-of-each-vertex/
 * Runtime: 3 ms
 * Memory: 18.4 MB
 * Solved via CodePath Auto-Committer
 */

#include <stdlib.h>

int* findDegrees(int** matrix, int matrixSize, int* matrixColSize, int* returnSize) {

    *returnSize = matrixSize;

    int *ans = (int *)malloc(matrixSize * sizeof(int));

    for (int i = 0; i < matrixSize; i++) {
        ans[i] = 0;

        for (int j = 0; j < matrixColSize[i]; j++) {
            ans[i] += matrix[i][j];
        }
    }

    return ans;
}

File: solutions/Question 1470/Solution1.py
# Problem: 1470. Shuffle the Array
# Difficulty: Easy
# Topics: Array
# URL: https://leetcode.com/problems/shuffle-the-array/
# Runtime: 58 ms
# Memory: 17.8 MB
# Solved via CodePath Auto-Committer

class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        x = []
        for a in range(n):
            x.append(nums[a])
            x.append(nums[a+n])
        return x